windows-gaming-driver/src/setup/usb.rs

```diff
@@ -82,18 +84,116 @@ pub fn select(machine: &mut MachineConfig) -> bool {
     true
 }
 
+/// Lets the user remove (some) of given devices and returns a tuple of the mouse and keyboard still in the list afterwards
+///
+/// This modifies the passed list.
+fn remove(usb_devices: &mut Vec<UsbBinding>) -> (Option<UsbBinding>, Option<UsbBinding>) {
+    let devs = list_devices(None).expect("Can't read connected USB devices");
+
+    loop {
+        println!("You have currently selected the following usb devices: ");
+        let mut last = 0;
+        for (i, binding) in usb_devices.iter().cloned().enumerate() {
+            if let Some(pos) = devs.iter().position(|dev| UsbBinding::from(dev.id) == binding
+                    || UsbBinding::from(dev.port) == binding) {
+                // device is connected
+                let dev = &devs[pos];
+                let name = to_name(dev.vendor_name.as_ref(), dev.product_name.as_ref());
+                println!("[{}]\t{} [{:04x}:{:04x}]", i, name, dev.id.vendor, dev.id.product);
```

Comment:
Consider printing bus+port instead of vendor+model here if the device is bound by port.